journeys/vikasganiga05.md

```diff
@@ -1,8 +1,8 @@
 # [Vikas Ganiga](https://github.com/vikasganiga05)'s Coding Journey
 
-Hi, I am a self-taught Full Stack Developer, Open Source, and Tech Enthusiast from India. I am primarily focused on JavaScript, but my learning style allows me to adapt to new languages and ideologies with ease. I've been building stuff on the web since 2019, I've made countless side projects. I like creating simple and effective UI that optimizes the user experience.
+[Hi, I am a self-taught Full Stack Developer, Open Source, and Tech Enthusiast from India. I am primarily focused on JavaScript, but my learning style allows me to adapt to new languages and ideologies with ease. I've been building stuff on the web since 2019, I've made countless side projects. I like creating simple and effective UI that optimizes the user experience.
```

Comment:
```suggestion
Hi, I am a self-taught Full Stack Developer, Open Source, and Tech Enthusiast from India. I am primarily focused on JavaScript, but my learning style allows me to adapt to new languages and ideologies with ease. I've been building stuff on the web since 2019. I've made countless side projects. I like creating simple and effective UI that optimizes the user experience.
```